doc/source/whatsnew/v0.23.0.txt

```diff
@@ -419,6 +419,7 @@ Datetimelike
 - Bug in ``.astype()`` to non-ns timedelta units would hold the incorrect dtype (:issue:`19176`, :issue:`19223`, :issue:`12425`)
 - Bug in subtracting :class:`Series` from ``NaT`` incorrectly returning ``NaT`` (:issue:`19158`)
 - Bug in :func:`Series.truncate` which raises ``TypeError`` with a monotonic ``PeriodIndex`` (:issue:`17717`)
+- Bug in :func:`NDFrame.pct_change` produces inconsistent frames using ``periods`` and ``freq`` (:issue:`7292`)
```

Comment:
This link to `NDFrame` won't work since it isn't in our API pages. Try

    :func:`~DataFrame.pct_change`

It's also not clear (to me) what "inconsistent frames" means here.

Reply:
added change in new commit.